src/app/pages/office/office.actions.ts

```diff
@@ -0,0 +1,29 @@
+import { createAction } from '@ngrx/store';
+
+export const TestStartDateChanged = createAction(
+  '[OfficePage] Test Start Date Changed',
+  (previousStartDate: string, customStartDate: string) => ({ previousStartDate, customStartDate }),
+);
+
+export const OfficeViewDidEnter = createAction(
+  '[OfficePage] Office view did enter',
+);
+
+export const CompleteTest = createAction(
+  '[OfficePage] Complete Test',
+);
+
+export const SavingWriteUpForLater = createAction(
+  '[OfficePage] Saving write-up for later',
+);
+
+export const OfficeValidationError = createAction(
+  '[OfficePage] Validation error',
+  (errorMessage: string) => ({ errorMessage }),
+);
+
+export type OfficeActionTypes =
```

Comment:
We can now use ```import { union } from '@ngrx/store';``` to achieve this.

Think this can be removed however as its unused id the ```officeReducer``` is un-needed

Reply:
unused, removed